chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -21,6 +21,8 @@
   include_recipe "bluepill"
 end
 
+include_recipe "barclamp::default"
```

Comment:
Im guessing you do this to have the `Barclamp` namespace available, but as far as I understod, just depending on it should make it available to the recipe no? its loaded on the context of the chef run for sure so Im not sure if we need this.

Reply:
True, this required a small change, but done.